internal/maintenance/periodic_job_enqueuer.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/riverqueue/river/rivertype"
 )
 
+const metadataKeyPeriodicJobID = "river:periodic_job_id"
```

Comment:
Thoughts on putting this const with other metadata keys in https://github.com/riverqueue/river/blob/ae827888c816384130a3a13dcd9f038d504fd167/internal/rivercommon/river_common.go#L23 or https://github.com/riverqueue/river/blob/ae827888c816384130a3a13dcd9f038d504fd167/rivertype/river_type.go#L14 ?